component/FloatingActionItem.js

```diff
@@ -140,8 +141,8 @@ const styles = StyleSheet.create({
     shadowRadius: 3,
     elevation: 5,
     borderRadius: 4,
-    height: 22,
-    marginTop: 8
+    marginTop: 4,
```

Comment:
I'm not a fan of changing this because everybody who is using this plugin will suffer from this changes... so we can add this property into variables